static/src/javascripts/projects/common/modules/loyalty/save-for-later.js

```diff
@@ -0,0 +1,90 @@
+define([
+    'qwery',
+    'bonzo',
+    'bean',
+    'common/utils/_',
+    'common/utils/config',
+    'common/utils/mediator',
+    'common/modules/identity/api'
+], function (
+    qwery,
+    bonzo,
+    bean,
+    _,
+    config,
+    mediator,
+    identity
+) {
+    function SaveForLater() {
+        this.saveLinkHolder = qwery('.meta__save-for-later')[0];
+        this.userData = null;
+        this.encodedPageUrl = encodeURIComponent(document.location.href);
+        this.$saver = bonzo(this.saveLinkHolder);
+        this.savedArticlesUrl = config.page.idUrl + '/saved-articles';
+    }
+
+    SaveForLater.prototype.init = function () {
+        if (identity.isUserLoggedIn()) {
+            this.getSavedArticles();
+        } else {
+            var url = config.page.idUrl + '/prefs/save-content?returnUrl=' + this.encodedPageUrl + '&shortUrl=' + config.page.shortUrl;
+            this.$saver.html(
+                '<a href="' + url + ' "data-link-name="meta-save-for-later" data-component=meta-save-for-later">Save for later</a>'
+            );
+        }
+    };
+
+    SaveForLater.prototype.getSavedArticles = function () {
+        var self = this,
+            notFound  = [{message:'Not found', description:'Resource not found'}];
+
+        identity.getUsersSavedDocuments().then(
+            function success(resp) {
+                if (resp.status === 'error') {
+                    if (JSON.stringify(notFound) ===  JSON.stringify(resp.errors)) {
+                        //Identity api needs a string in the format yyyy-mm-ddThh:mm:ss+hh:mm  otherwise it barfs
+                        var date = new Date().toISOString().replace(/\.[0-9]+Z/, '+00:00');
```

Comment:
@NathanielBennett Interesting, I saw `moment` in `package.json` and I thought we were using it already. But it's not in `requirejs.js` and I can't find any place where we actually use it. Probably a forgotten dependency. Fine